client/knowledge_base_search/src/containers/TrainingContainer/TrainingComparison.jsx

```diff
@@ -2,20 +2,28 @@ import React, { Component } from 'react';
 import { string, number, shape } from 'prop-types';
 import ResultContainer from '../ResultContainer/ResultContainer';
 import replaceNewlines from '../../utils/replaceNewlines';
+import { Icon } from 'watson-react-components';
+import classNames from 'classnames';
 import './styles.css';
 
 class TrainingComparison extends Component {
-  getRankDisplay() {
-    const { trainedResult: { originalRank } } = this.props;
 
-    if (originalRank === 0) {
-      return 'Untrained Rank: > 100';
+  getRankDisplay(trainingRank, originalRank) {
+    if (originalRank && trainingRank !== originalRank) {
+      const relativeRank = trainingRank < originalRank ? 'up' : 'down';
+      return (
+        <div className={classNames('training_comparison--rank-'+relativeRank)}>
```

Comment:
The way that this is currently written, you're not using the `classnames` API correctly. This is actually just concatenating two strings together, something that you get with just plain ol' javascript.

For example, you could drop the classnames usage and just do 

```js
className={'training_comparison--rank-' + relativeRank}
```

or 

```js
className={`training_comparison--rank-'${relativeRank}`}
```

If you want to use the classnames API, this should be re-written to something like

```js
const relativeRank = classNames({
  'training_comparison--rank-up': trainingRank < originalRank
  'training_comparison--rank-down': trainingRank > originalRank 
});

<div className={relativeRank}>
```